systemlock-0.1/debian/control

```diff
@@ -10,3 +10,8 @@ Package: systemlock
 Architecture: all
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: Interface for ofris.
+ .
```

Comment:
This first line is not required;
Add one space indentation in the second line 
The last line is not required; the second line is sufficient